### PR TITLE
[WIP] Extract BCryptImportKey from Cng.cs to Interop.BCryptImportKey.cs

### PR DESCRIPTION
**Pull request by robert-matusewicz**
Extract BCryptImportKey from Cng.cs to Interop.BCryptImportKey.cs

### CODE REVIEW

File: src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            public uint dwMagic;
+            public uint dwVersion;
+            public uint cbKeyData;
+
+            public const uint BCRYPT_KEY_DATA_BLOB_MAGIC = 0x4d42444b;
+            public const uint BCRYPT_KEY_DATA_BLOB_VERSION1 = 0x1;
+        }
+        
+        [DllImport(Libraries.BCrypt, CharSet = CharSet.Unicode)]
+         public static extern NTSTATUS BCryptImportKey(SafeAlgorithmHandle hAlgorithm, IntPtr hImportKey, string pszBlobType, out SafeKeyHandle hKey, IntPtr pbKeyObject, int cbKeyObject, byte[] pbInput, int cbInput, int dwFlags);
```
Comment (bartonjs) [inline src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs]: Nit: Something seems to have gotten misaligned here, whitespace-wise

Comment (bartonjs) [inline src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs]: If the only caller of this is the helper method, please change the access level here to private.

File: src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
```
Comment (bartonjs) [inline src/Common/src/Interop/Windows/BCrypt/Interop.BCryptImportKey.cs]: FYI: You seem to be missing .csproj edits that would add this file to the compilation.

**Comment by maryamariyan** [comment]
Thank you for your contribution. As announced in dotnet/coreclr#27549 this repository will be moving to dotnet/runtime on November 13. If you would like to continue working on this PR after this date, the easiest way to move the change to dotnet/runtime is:
1.	In your corefx repository clone, create patch by running `git format-patch origin`
2.	In your runtime repository clone, apply the patch by running `git apply --directory src/corefx <path to the patch created in step 1>`

**Pull request closed without merging**